### DISCUSSION TITLE
https://mnfienen.github.io/talulat/2023/08/16/day15.html?fbclid=IwAR3YpEXagNNn3MMWceR54pOtUhkB2IqgqY90guKLWBxXa5F3UR-_2AyJBZQ_aem_AS30taDdu44FugLzUo6QcnneW6lVqoUrwpP6cgqhwXtsHKpFW3J7xmTRQOJ56sG810c

### DISCUSSION THREAD

**Discussion by giscus** [mnfienen/talulat#23] [Comments]
# https://mnfienen.github.io/talulat/2023/08/16/day15.html?fbclid=IwAR3YpEXagNNn3MMWceR54pOtUhkB2IqgqY90guKLWBxXa5F3UR-_2AyJBZQ_aem_AS30taDdu44FugLzUo6QcnneW6lVqoUrwpP6cgqhwXtsHKpFW3J7xmTRQOJ56sG810c

Day 15 - To the north shore! miles today: 86 total miles: 947 km today: 139 total km: 1525

https://mnfienen.github.io/talulat/2023/08/16/day15.html?fbclid=IwAR3YpEXagNNn3MMWceR54pOtUhkB2IqgqY90guKLWBxXa5F3UR-_2AyJBZQ_aem_AS30taDdu44FugLzUo6QcnneW6lVqoUrwpP6cgqhwXtsHKpFW3J7xmTRQOJ56sG810c

**Comment by SmittysGitHub** [comment]
Your adventure is incredible.

**Comment by aleaf** [comment]
sounds like a pretty rad day!
